### PR TITLE
Fix redirect login

### PR DESCRIPTION
**Pull request by paulotijero**

### CODE REVIEW

File: src/reducer.js [hunk 1]
```diff
@@ -59,11 +59,9 @@ function reducer(state = initialState, action = {}) {
     }
 
     case "LIST_RESTAURANTS": {
-      console.log(action.payload);
-
       return {
         ...state,
-        restaurants: [...action.payload]
+        restaurants: action.payload
```
Comment (stg101) [inline src/reducer.js]: Why you have change this line?

File: src/views/home.js [hunk 2]
```diff
@@ -14,12 +16,15 @@ const container = {
...
 
 function Home() {
+  const user = useUser();
   const listRestaurants = useListRestaurants();
 
   React.useEffect(() => {
     listRestaurants();
   }, []);
 
+  if (!user) return <Redirect to="/login" noThrow />;
```
Comment (stg101) [inline src/views/home.js]: What does this line does?

Comment (paulotijero) [reply src/views/home.js]: This line verify if the user is logged in in the page.

Comment (stg101) [reply src/views/home.js]: Thanks !

File: src/reducer.js (hunk 1 above)
Comment (paulotijero) [reply src/reducer.js]: Because they don't work like that. I understand that the action doesn't edit, maybe @DiegoCuevas  can explain more about this.

**Pull request merged**